adafruit_ht16k33/ht16k33.py

```diff
@@ -43,13 +43,19 @@ class HT16K33:
     def __init__(
         self,
         i2c: I2C,
-        address: int = 0x70,
+        address: Union[int, Tuple, List] = 0x70,
         auto_write: bool = True,
         brightness: float = 1.0,
     ) -> None:
-        self.i2c_device = i2c_device.I2CDevice(i2c, address)
+        if isinstance(address, (tuple, list)):
+            self.i2c_device = []
+            for addr in address:
+                self.i2c_device.append(i2c_device.I2CDevice(i2c, addr))
```

Comment:
This is a neat solution.  The only thing that comes to mind is that this technically now means that this library wouldn't work with `adafruit_register` but that doesn't apply here.

Reply:
Thanks, yeah.